### ISSUE TITLE
Import fails if Array.prototype contains unexpected enumerable properties

### ISSUE THREAD

**Issue by jfn4th** [fryguy1013/pdftofoundry#3]
Tested on Foundry 306, PF2e 5.2.3
Error is new to 4.1.1, does not occur in 4.1.0

If I'm not mistaken, Foundry v11 no longer adds to Array.prototype, though some other modules add their own properties. In my case Warp Gate is adding a method called removeIf, though anything as simple as `Array.prototype.foo = 1` is enough to break things. For now, simple enough for me to just disable other modules during import.

Here's the console output
```
Uncaught (in promise) _UnknownErrorException {message: 'The `Array.prototype` contains unexpected enumerab…s breaking e.g. `for...in` iteration of `Array`s.', name: 'UnknownErrorException',
 details: 'Error: The `Array.prototype` contains unexpected e…s breaking e.g. `for...in` iteration of `Array`s.'}details: "Error: The `Array.prototype` contains unexpected enumerable properties: removeIf; thus breaking e.g. `for...in` iteration of `Array`s."
 message: "The `Array.prototype` contains unexpected enumerable properties: removeIf; thus breaking e.g. `for...in` iteration of `Array`s."
 name: "UnknownErrorException"
[[Prototype]]: Error
generateData @ importer.mjs:1
await in generateData (async)
callback @ FoundryGui.mjs:1
await in callback (async)
submit @ foundry.js:64992
_onClickButton @ foundry.js:64920
dispatch @ jquery.min.js:2
y.handle @ jquery.min.js:2
```

**Comment by fryguy1013** [comment]
What adventure are you trying to import?

**Comment by jfn4th** [comment]
PFS #2-09 is what I was trying when I first had the issue. Tried out a few other things (Troubles in Otari, Quest for the Frozen Flame #1, a few others as well) and got the same error each time.

**Comment by fryguy1013** [comment]
Should be fixed in 4.1.2

**Comment by thebrownjumper** [comment]
Getting the same when trying to import Troubles in Otari.

<img width="412" alt="PDF Importer" src="https://github.com/fryguy1013/pdftofoundry/assets/81319107/f3377c34-6dde-4014-a584-d37b426371df">

When is 4.1.2 being released?

**edit** Oops read (properly) the above by @jfn4th -  disabled all modules and its all imported fine.
